pom.xml

```diff
@@ -502,6 +508,41 @@
 				</plugins>
 			</build>
 		</profile>
+		
+		<profile>
+			<id>nonRelease</id>
```

Comment:
Rename to `snapshot`?

Reply:
sure! I will change and re-push.

Reply:
hang on a minute - no, better not - this profile isn't for snapshots. It's for *non-release builds* - i.e. builds not performed by the release plugin. In fact we explicitly want this *for non-snapshots*.